qmxgraph/page/api.js

```diff
@@ -410,6 +411,24 @@ graphs.Api.prototype.getCellIdAt = function getCellIdAt (x, y) {
     return !!cell? cell.getId() : null;
 };
 
+/**
+ * Return the id of the edge containing the given decoration.
+ *
+ * @param cellId
+ * @returns {number} Id of the decoration's parent.
+ * @throws {Error} If the given cell is not found or it is a decoration.
+ */
+graphs.Api.prototype.getDecorationParentCellId = function getParentCellId (cellId) {
```

Comment:
Please use `getParentCellId` for the name of the function to match the prototype